Source/WebCore/html/shadow/attachmentElementShadow.css

```
@@ -170,6 +199,17 @@ button#attachment-save-button {
 }
 
 button#attachment-save-button::before {
+#if (defined(WTF_PLATFORM_IOS_FAMILY) && WTF_PLATFORM_IOS_FAMILY)
+#if (defined(WTF_PLATFORM_VISION) && WTF_PLATFORM_VISION)
+    mask-size: 14px;
+#else
+    mask-size: 18px;
+#endif
+    background-color: -apple-system-blue;
+#else
+    mask-size: 14px;
+    background-color: -apple-system-secondary-label;
+#endif
```

```suggestion
#if (defined(WTF_PLATFORM_IOS_FAMILY) && WTF_PLATFORM_IOS_FAMILY)
    background-color: -apple-system-blue;
#else
    background-color: -apple-system-secondary-label;
#endif
#if (defined(WTF_PLATFORM_IOS) && WTF_PLATFORM_IOS))
    mask-size: 18px;
#else
    mask-size: 14px;
#endif
```

Do you know if this works?

---

I think it would, but I personally think what I have is more readable imo since it's more explicit about having vision specific things, since it's just a coincidence that macOS and visionOS have the same size.